eve/methods/get.py

```diff
@@ -184,7 +183,7 @@ def prune_aggregation_stage(d):
             abort(400, description="Aggregation query could not be parsed.")
 
         for key, value in query.items():
-            if key[0] != "$":
+            if key.startswith("$"):
```

Comment:
Was this change intended or should it be `if not key.startswith("$"):`

Reply:
Hmm not intended.

Reply:
I guess this code path isn't being tested?

Reply:
Thanks for the review @mic-max. Feel free to PR a fix.